Carthage.md

```diff
@@ -0,0 +1,61 @@
+# Firebase Carthage
+
+## Context
+
+This page introduces and provides instructions for an **experimental** Firebase
+[Carthage](https://github.com/Carthage/Carthage) implementation. Based on
+feedback and usage, the Firebase team may decide to make to make the Carthage
+distribution official.
+
+## Carthage Installation
+
+[Homebrew](http://brew.sh/) is one way to install Carthage.
+
+```bash
+$ brew update
+$ brew install carthage
+```
+
+See the
+[Carthage page](https://github.com/Carthage/Carthage#installing-carthage) for
+more details and additional installation methods.
+
+## Carthage Usage
+
+- Create a Cartfile with a subset of the following components. Note that
+    **FirebaseAnalytics** must always be included.
+    - binary "https://dl.google.com/dl/firebase/ios/carthage/FirebaseABTestingBinary.json"
+    - binary "https://dl.google.com/dl/firebase/ios/carthage/FirebaseAdMobBinary.json"
+    - binary "https://dl.google.com/dl/firebase/ios/carthage/FirebaseAnalyticsBinary.json"
+    - binary "https://dl.google.com/dl/firebase/ios/carthage/FirebaseAuthBinary.json"
+    - binary "https://dl.google.com/dl/firebase/ios/carthage/FirebaseCrashBinary.json"
+    - binary "https://dl.google.com/dl/firebase/ios/carthage/FirebaseDatabaseBinary.json"
+    - binary "https://dl.google.com/dl/firebase/ios/carthage/FirebaseDynamicLinksBinary.json"
+    - binary "https://dl.google.com/dl/firebase/ios/carthage/FirebaseFirestoreBinary.json"
+    - binary "https://dl.google.com/dl/firebase/ios/carthage/FirebaseInvitesBinary.json"
+    - binary "https://dl.google.com/dl/firebase/ios/carthage/FirebaseMessagingBinary.json"
+    - binary "https://dl.google.com/dl/firebase/ios/carthage/FirebasePerformanceBinary.json"
+    - binary "https://dl.google.com/dl/firebase/ios/carthage/FirebaseRemoteConfigBinary.json"
+    - binary "https://dl.google.com/dl/firebase/ios/carthage/FirebaseStorageBinary.json"
+- Run `carthage update`
+- Use the Finder to open `Carthage/Build/iOS`.
```

Comment:
nit: Apple references Finder without a prepended `the`.

`Use Finder to open ...`